pkg/network/server_test.go

```diff
@@ -87,20 +87,15 @@ func TestNewServer(t *testing.T) {
 	})
 }
 
-func startWithChannel(s *Server) chan error {
-	ch := make(chan error)
-	go func() {
-		s.Start(ch)
-		close(ch)
-	}()
-	return ch
+func start(s *Server) {
```

Comment:
Well, we may drop this one-line wrapper too.

Reply:
Fixed.